demisto_sdk/common/hook_validations/base_validator.py

```diff
@@ -0,0 +1,105 @@
+import os
+import re
+from abc import abstractmethod
+
+from demisto_sdk.common.constants import Errors
+from demisto_sdk.common.hook_validations.structure import StructureValidator
+from demisto_sdk.common.tools import print_error, get_release_notes_file_path, \
+    get_latest_release_notes_text, run_command
+
+
+class BaseValidator(object):
+    DEFAULT_VERSION = -1
+
+    def __init__(self, structure_validator):
```

Comment:
this is python 3 now, types should be on the arguments
add docstrings

Reply:
it's valid also fro py3